pkg/config/env.go

```diff
@@ -30,6 +30,11 @@ var Config = struct {
 	EvalCacheRefreshTimeout time.Duration `env:"FLAGR_EVALCACHE_REFRESHTIMEOUT" envDefault:"59s"`
 	// EvalCacheRefreshInterval - time interval of getting the flags data from DB into the in-memory evaluation cache
 	EvalCacheRefreshInterval time.Duration `env:"FLAGR_EVALCACHE_REFRESHINTERVAL" envDefault:"3s"`
+	// EvalEnableDebug - controls if we want to return evaluation debugging information back to the api requests
+	// Note that this is a global switch:
+	//     if it's disabled, no evaluation debug info will be returned.
+	//     if it's enabled, it respects evaluation request's enableDebug field
+	EvalDebugEnabled bool `env:"FLAGR_EVAL_DEBUG_ENABLED" envDefault:"true"`
```

Comment:
True as default? I would make it less intrusive ...

Reply:
True is the current behavior. It controls globally if the server will need to respect the "enableDebug" filed from the request. https://checkr.github.io/flagr/api_docs/#operation/postEvaluation

Reply:
I wasn't aware, good to know 😄